aya-log/src/lib.rs

```diff
@@ -326,6 +331,9 @@ impl Log for DefaultLogger {
 
 #[derive(Error, Debug)]
 pub enum Error {
+    #[error("log event array {} doesn't exist", MAP_NAME)]
+    MapMissingError,
```

Comment:
I think this should be `MapNotFound`.

We use the error suffix when forwarding an inner error, eg `MapError(map_error)` `IOError(io_error)` etc.

Reply:
Done.